core/src/CMakeLists.txt

```diff
@@ -224,7 +224,9 @@ if(APPLE)
         target_link_libraries(ledger-core-interface INTERFACE -static-libstdc++)
     endif()
     target_link_libraries(ledger-core-interface INTERFACE Threads::Threads)
-    target_link_libraries(ledger-core-obj PUBLIC Threads::Threads)
+    if (TARGET ledger-core-obj)
+        target_link_libraries(ledger-core-obj PUBLIC Threads::Threads)
```

Comment:
You can move this line up, in the correct block. Like at line 101.

Reply:
Threads is required only in APPLE block. So there is no way to make this shorter.

Reply:
Alright !

Reply:
Alright !